app/src/pages/inside/testsPage/historyLine/historyLineItem/historyLineItemContent/historyLineItemBadges/historyLineItemBadges.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import {
+  AUTOMATION_BUG,
+  NO_DEFECT,
+  PRODUCT_BUG,
+  SYSTEM_ISSUE,
+  TO_INVESTIGATE,
+} from 'common/constants/defectTypes';
+import { SKIPPED, RESETED, FAILED, MANY, NOT_FOUND } from 'common/constants/launchStatuses';
+import Parser from 'html-react-parser';
+import styles from './historyLineItemBadges.scss';
+import NoItemIcon from './img/noItem-inline.svg';
+import EmptyItemIcon from './img/emptyItem-inline.svg';
+import NotEyeIcon from './img/notEyeItem-inline.svg';
+import DurationIcon from './img/duration-icon-inline.svg';
+import InfoIcon from './img/info-icon-inline.svg';
```

Comment:
Please check info icon. You don't 'inform' icon.

Reply:
Fixed